maven-model/src/test/java/org/apache/maven/model/OrganizationTest.java

```diff
@@ -53,4 +53,36 @@ public void testToStringNullSafe()
         assertNotNull( new Organization().toString() );
     }
 
+
+    public void testToStringNotNonsense11()
+    {
+        Organization org = new Organization();
+        org.setName( "Testing Maven Unit" );
+        org.setUrl( "https://maven.localdomain" );
+
+        assertEquals( "name = 'Testing Maven Unit'\nurl = 'https://maven.localdomain'", org.toString() );
+    }
+
+    public void testToStringNotNonsense10()
+    {
+        Organization org = new Organization();
+        org.setName( "Testing Maven Unit" );
+
+        assertEquals( "name = 'Testing Maven Unit'\nurl = 'null'", org.toString() );
```

Comment:
The `'null'` definitely looks wrong.  If quotes are added when there is a string, it should be `url = null` when the url is actually `null` (with no quotes).

Reply:
I just let modello do that.